src/main/java/com/rachelbock/resources/ProjectResource.java

```diff
@@ -0,0 +1,127 @@
+package com.rachelbock.resources;
+
+import com.rachelbock.data.Climb;
+import com.rachelbock.data.Project;
+import com.rachelbock.data.User;
+
+import javax.swing.plaf.nimbus.State;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Properties;
+
+/**
+ * Class to set up the database connection for projects
+ */
+
+@Path("/projects/{user_name}")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class ProjectResource {
+
+    public Connection getConnection() throws SQLException {
+        Properties connectionProps = new Properties();
+        connectionProps.put("user", "root");
+        connectionProps.put("password", "root");
+
+        String dbms = "mysql";
+        String serverName = "localhost";
+        int portNumber = 3306;
+        String dbName = "clamber";
+
+        String connectionString = "jdbc:" + dbms + "://" + serverName + ":" + portNumber + "/" + dbName;
+        Connection conn = DriverManager.getConnection(connectionString, connectionProps);
+        return conn;
+    }
+
+    @GET
```

Comment:
Probably a good idea to start adding javadocs to the GET methods, though these are fairly straightforward. Something along the lines of "Retrieves climb data for all climbs that the given user has marked as being a project. Returns an empty list if that user has no projects, or if the user does not exist." should be sufficient. The idea being to make it clear what data it's returning, and how it handles edge cases.